conf/modules.config

```diff
@@ -197,6 +232,7 @@ process {
     }
 
     withName: KRAKEN2 {
+        ext.prefix = { ${meta.id} }
```

Comment:
This is same as the prefix assigned by the module, right? Did you mean to assign a different prefix?

Reply:
I think it was a copy paste error :grimacing: